src/App.js

```diff
@@ -1,18 +1,16 @@
 import React, { useState, useEffect } from 'react';
 import { Route, Link, Switch } from 'react-router-dom';
 import { db } from './lib/firebase.js';
-import { writeToFirestore } from './lib/firebase';
 import './App.css';
 import List from './components/List';
 import AddItem from './components/AddItem';
 import BottomNav from './components/BottomNav';
 
 function App() {
   let [results, setResults] = useState([]);
-  let [name, setName] = useState('');
 
   useEffect(() => {
-    db.collection('shopping-list').onSnapshot(function(querySnapshot) {
+    db.collection('put-token-here').onSnapshot(function(querySnapshot) {
```

Comment:
I noticed in #3 we moved this useEffect call into the List component itself. I'm wondering whether we should keep the call in App.js just like you have here when we resolve merge conflicts as the entire app will depend on this data on initial load.

Reply:
Maybe we can do a group merge since this one has a lot of overlap!